src/components/element-editor/validation/validation-child.tsx

```diff
@@ -0,0 +1,96 @@
+import { Form } from "@lifesg/react-design-system";
+import { useEffect, useState } from "react";
+import { ChildEntry } from "src/components/common";
+import { IValidation } from "src/context-providers/builder";
+import { FieldWrapper } from "./validation.styles";
+
+interface IProps {
+    onDelete: () => void;
+    options: string[];
+    onChange: (newValue: any) => void;
+    value?: IValidation;
+}
+
+export const ValidationChild = ({
+    onDelete,
+    options,
+    onChange,
+    value,
+}: IProps) => {
+    // =========================================================================
+    // CONST, STATE, REF
+    // =========================================================================
+    const [type, setType] = useState<string>();
+
+    // =============================================================================
+    // EVENT HANLDERS
+    // =============================================================================
+    const handleChange = (changeType?: string, newValue?: string) => {
+        const updatedValue = { ...value };
+        if (!updatedValue.validationType || options.length === 1) {
```

Comment:
this wont work if there are multiple validation types, why not use just set the type directly when `Form.Select` has a change in selected option?